docs/source/installation.md

```diff
@@ -125,7 +125,7 @@ You can install `vllm` and `vllm-ascend` from **pre-built wheel** (**Unreleased
    :substitutions:
 
 # Install vllm-project/vllm from pypi
-pip install vllm==|pip_vllm_version|
+VLLM_TARGET_DEVICE=empty pip install vllm==|pip_vllm_version|
```

Comment:
I a little bit forgot why we need empty? is this for x86?

Should we also change this in workflow? Does main need it?

Reply:
$ VLLM_TARGET_DEVICE=empty pip install vllm==0.7.3
// ... ...
Requested vllm==0.7.3 from https://mirrors.tuna.tsinghua.edu.cn/pypi/web/packages/f5/73/87825ee293f0e721eb8aadbd0cf9c3436d4e3b3458322878f1f6e7398a0d/vllm-0.7.3.tar.gz has inconsistent version: expected '0.7.3', but metadata has '0.7.3+empty'
ERROR: No matching distribution found for vllm==0.7.3